tools/linter/clang_tidy/requirements.txt

```diff
@@ -0,0 +1 @@
+unidiff==0.6.0
```

Comment:
For a follow up: If we make the codebase clang-tidy clean and remove the line filter I don't think we'd need this anymore since it's pretty easy to manually parse out filenames from a diff file (as opposed to parsing names + hunk line ranges which isn't as straightforward)